src/Pages/HomePage/Carousel/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import './Carousel.css';
+import Card from './Card';
+
+
+class Carousel extends React.Component{
+    constructor(props){
+        super(props)
+        this.state={
+            prod : [],
+            left : 0,
+        }
+        this.prevSlide = this.prevSlide.bind(this);
```

Comment:
do not use bind at all convert all the handlers into arrow function, after that there is no need of bind